README.md

```diff
@@ -499,3 +499,14 @@ grep -r -l "smt" . | xargs sed -i 's/smt/my-package/'
 #### How can I create benchmarks for my library?
 
 You can see some examples in the `benchmarks` folder. All you have to do is create a file that exports a function to run your benchmark in that folder, and add that function to the `index.ts` file. The `yarn benchmarks` command can be run with no parameters (it will run all the benchmarks), or you can specify the name of your benchmark file to run just that. When you run the command it will create a `benchmarks/results` folder with your results.
+
+#### I need to use a Merkle Tree to prove the inclusion or exclusion of data elements within a set. Which type of Merkle Tree should I use?
+**Incremental:** Ideal for applications where you frequently add new elements and need to update the tree efficiently.
+***Lean Incremental:** A more memory-efficient version of the incremental Merkle tree, suitable for use cases with memory constraints.
+**Sparse:** Best suited for scenarios where you need to manage a large, mostly empty set of possible elements.
+
+| Type            | Library Name              | Main Feature                                 | Recommended for                              | Not Recommended for                         | Used by                                       | Proof Gen Speed       |
+|-----------------|---------------------------|----------------------------------------------|-----------------------------------------------|----------------------------------------------|------------------------------------------------|-----------------------|
+|**Incremental**  | @zk-kit/imt               | Fastest for incremental updates              | Applications requiring frequent updates       | Scenarios with very large datasets           | Semaphore, Tornado Cash                        | Fast                  |
+|**Lean incremental**| @zk-kit/lean-imt          | Optimized for lightweight environments       | Mobile and browser-based ZK applications      | High-frequency update requirements           | zkSync, Loopring                                | Moderate              |
+|**Sparse**           | @zk-kit/smt               | Handles very large sets efficiently          | Applications with large static datasets       | Frequent updates due to higher computational cost | Aztec, Zcash                   | Slow (due to large set handling) |
```

Comment:
what about adding links to the homepage of the projects listed?

Reply:
Got it, working on it